TUnit.Templates/content/TUnit.AspNet/TestProject/Tests.cs

```diff
@@ -1,5 +1,7 @@
 ﻿namespace TestProject;
 
+[assembly: System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
```

Comment:
Could we split this out into a GlobalSetup file again just for separation of concerns?

Reply:
Fixed!